util/xsl/695.par.xsl

```diff
@@ -0,0 +1,157 @@
+<!--  This xsl:
+    * for simplified generators (PV,PQ)
+        - replaces the AlphaPu by an AlphaPuPNom (= AlphaPu * SnRef / PNom)
+        - creates a PNom parameter
+        - replaces PMinPu and PMaxPu parameters by references to IIDM values
+    * for PV generators only:
+        - replaces the LambdaPu by a LambdaPuSNom (= LambdaPu * SnRef / SNom)
+        - creates a SNom parameter
+        - replaces QMinPu and QMaxPu by QMin and QMax (= QMin * SnRef)
+-->
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:dyn="http://www.rte-france.com/dynawo">
+
+<!--  These lines enable to avoid empty lines after removing some elements while keeping the indentation fine -->
+<xsl:output omit-xml-declaration="yes" indent="yes"/>
+<xsl:strip-space elements="*"/>
+
+<!--  This first template copies all the xml. It will be overrided by the following templates when they could be applied -->
+<xsl:template match="@*|node()">
+  <xsl:copy>
+    <xsl:apply-templates select="@*|node()"/>
+  </xsl:copy>
+</xsl:template>
+
+<!--  To deal with file containing dynawo namespace -->
+<!--  We go through all the par entries of each set: <par type="" ... />
```

Comment:
Thank you for this very detailed xsl, the way it is implemented is clear and the comment are very helpful!